macros/jobOptions.mac

```diff
@@ -12,8 +12,8 @@
 #
 #/WCSim/physics/list QGSP_BIC_HP #preferred for energies below 5 GeV
 #/WCSim/physics/list LBE # preferred for low-background experiments.  
-/WCSim/physics/list FTFP_BERT # recommended for high energy physics and hadron production up to beam momentum of several tens of GeV/c.
-#/WCSim/physics/list FTFP_BERT_HP
-/WCSim/physics/nCapture Default
+#/WCSim/physics/list FTFP_BERT # recommended for high energy physics and hadron production up to beam momentum of several tens of GeV/c.
+/WCSim/physics/list FTFP_BERT_HP
+#/WCSim/physics/nCapture Default
```

Comment:
Please revert this. If you'd like to modify the default physics option for WCTE, please submit a new PR that solves #422

Reply:
👍